specification/iotcentral/data-plane/readme.md

```diff
@@ -14,9 +14,9 @@ To see additional help and options, run:
 ``` yaml
 directive:
     - suppress: ValidFormats
-        from: iotcentral.json
-        where: $.definitions.EmailUser.allOf[1].properties.email.format
-        reason: email format is allowed
+      from: iotcentral.json
```

Comment:
Your spacing here is incorrect, causing the linter to fail.  Please see an example https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md

Reply:
This has been addressed thanks 👍